Sources/Disposable.swift

```diff
@@ -163,12 +159,10 @@ public final class CompositeDisposable: Disposable {
 	}
 
 	public func dispose() {
-		if state.tryDispose() {
-			if let ds = disposables.swap(nil) {
-				for d in ds {
-					d.dispose()
-				}
-			}
+		guard state.tryDispose(), let disposables = disposables.swap(nil) else { return }
+
+		for disposable in disposables {
+			disposable.dispose()
```

Comment:
Same here: I think `if` is clearer than `guard`.

Reply:
I prefer `guard`.
However, `swap` may be better to use` if`.